src/conditional-flow/multiple-conditions.js

```diff
@@ -29,7 +33,17 @@ function isHelloOrGoodbye (val1) {
 // 13-19  | Teenager
 // 20+    | Adult
 function getAgeDescription (age) {
-
+if (age === 0) {
+  return "Baby" 
+  } else if (1<=age&&age<=4){
```

Comment:
This might just be a personal thing for me, but I find it harder to read conditions laid out like `1<=age`. I think it's easier to read when you set them out such as `age >= 1`

Reply:
That makes sense, initially I tried ```1<=age<=4``` but this always seems to evaluate to true?